### PR TITLE
(feat): pre auth form

### PR DESCRIPTION
**Pull request by amosmachora**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary
This PR adds a pre-auth dashboard to view dashboard requests with a `make pre-auth request` form.

## Screenshots

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

File: packages/esm-billing-app/src/hooks/use-pre-auth-requests.ts [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+  profile: string[];
+}
+
+export interface Payee {
+  type: Priority;
+}
+
+export const usePreAuthRequests = (patientUuid: string) => {
+  const { mflCodeValue } = useSystemSetting('facility.mflcode');
+  const url = `https://payers.apeiro-digital.com/api/v1/claim/byFacility?facilityCode=Organization/${mflCodeValue}&type=preauthorization&claimResponseId=`;
```
Comment (Omoshlawi) [inline packages/esm-billing-app/src/hooks/use-pre-auth-requests.ts:171]: Kindly use config `hieBaseUrl`

`const url = `${hieBaseUrl}claim/byFacility?facilityCode=Organization/${mflCodeValue}&type=preauthorization&claimResponseId=``

File: packages/esm-billing-app/src/hooks/useInterventions.ts [hunk 2]
```diff
@@ -1,18 +1,13 @@
...
 import useSWR from 'swr';
-import { BillingConfig } from '../config-schema';
 import { SHAIntervension } from '../types';
 
-const useInterventions = (code: string) => {
-  const { hieBaseUrl } = useConfig<BillingConfig>();
-
-  const url = `${hieBaseUrl}/benefit-master/code?searchKey=${code}`;
+export const useInterventions = (code: string) => {
+  const url = `https://payers.apeiro-digital.com/api/v1/master/benefit-master/code?searchKey=${code}`;
```
Comment (Omoshlawi) [inline packages/esm-billing-app/src/hooks/useInterventions.ts:6]: same here

**Review by Omoshlawi — CHANGES_REQUESTED** [review]
Thankz @amosmachora , Left some suggestion

**Review by patryllus — APPROVED** [review]
Merging for tests. Good work

**Comment by ojwanganto** [comment]
A screenshot or short video should also be appropriate

**Comment by amosmachora** [comment]
> A screenshot or short video should also be appropriate

This was done in quite a hurry. Here is a screenshot of the form.
image

**Pull request merged**